packages/hint-leading-dot-classlist/src/hint.ts

```diff
@@ -36,7 +36,9 @@ export default class LeadingDotClasslist implements IHint {
                         return;
                     }
 
-                    if (node.callee.property.name !== 'add' && node.callee.property.name !== 'remove' && node.callee.property?.name !== 'classList') {
+                    if (!(node.callee.property.name === 'add' ||
+                    node.callee.property.name === 'remove')
+                    ) {
```

Comment:
Nit: Move this back to one line or add some to the second line.
```suggestion
                    if (!(node.callee.property.name === 'add' || node.callee.property.name === 'remove')) {
```